consul/data_source_consul_peerings.go

```diff
@@ -0,0 +1,123 @@
+package consul
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceConsulPeerings() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceConsulPeeringsRead,
+		Schema: map[string]*schema.Schema{
+			"datacenter": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Same as previous comment: datacenter isn't currently applicable to peerings.

Also the same two extra fields are also present in the response